README.md

```diff
@@ -26,37 +26,38 @@ An [OpenAPI specification](https://github.com/OAI/OpenAPI-Specification) describ
 
 A Looker SDK has several parts:
 
-- **Looker API** OpenAPI specification (e.g., found at
-  `https://<your-looker-endpoint>:19999/api/3.1/swagger.json`, although this is still the Swagger 2.x representation)
+- The **Looker API**, described by an OpenAPI specification (e.g., the Swagger 2.x representation found at
+  `https://<your-looker-domain>:19999/api/4.0/swagger.json`). The 4.0 API is our current & stable API. As of June 2022, [3.x is deprecated](https://developers.looker.com/api/advanced-usage/version-3x-deprecation).
 
 - The **Looker API Docs viewer**, provided in the Looker web app directly from our version-specific OpenAPI specification, available on each Looker server instance.
 
 - **Language SDKs**, "smarter" client language classes and methods to improve the experience of calling the Looker API in various popular coding languages.
 
-## Multi-API support with Looker 7.2 and later
+## SDK multi-API-version support
 
-Looker 7.2 introduced an **Experimental** version of API 4.0. Since that release, some SDKs now support multiple API versions in the same SDK package.
+The 4.0 version of the API is the current and stable version of the API, in addition to the 3.x API which is now [deprecated](https://developers.looker.com/api/advanced-usage/version-3x-deprecation).
 
-For all SDKs but Swift, API-specific SDKs are now created and put in the same SDK package, and share the same run-time code.
+Many SDKs support and expose both API versions in the same SDK package, including all [Looker-supported](https://docs.looker.com/reference/api-and-integration/api-sdk-support-policy#looker_supported) SDKs other than Swift.
```

Comment:
I'm pretty sure we've dropped 3.1 from Kotlin and Go as well. We should also drop it from Look# if we haven't yet because no one should use API 3.x on a statically typed language.

Reply:
I've cleaned this up a bit by focusing the language primarily around the "Looker supported" SDKs